examples/vault-prod-ready/README.md

```diff
@@ -0,0 +1,180 @@
+# Vault auto unseal cluster private with kms and acl for consul with encryption. PROD READY
+
+This example icombines the followings: Vault auto unseal - Vault private cluster - Consul with encryption and also adds some stuff that we expect in a prod ready working state, mainly:
+* Secure instances using security groups and remove [0.0.0.0/0]
+* Enable ACL on consul and pass the token in a secure way using AWS Secrets manager
+* Enable encryption on Consul (based on https://github.com/hashicorp/terraform-aws-consul/tree/master/examples/example-with-encryption)
+* Auto generate gossip_encryption_key and store it in AWS Secrets manager
+* Only allow ssh from Vpn server
+* Creates an ELB in front of Consul instances
+* Creates an ELB in front of Vault instances
+* Removed the block for deploying cluster in default VPC and availability zones, this needs to be set explicitly to enforce security
+
+This example use Terragrunt to solve some dependencies but can by used with terraform also.
+
+Thanks @mkrull for the ACL token provision
+
+# HOWTO
+* 1 - Run terragrunt
+* 2 - run `vault operator init` on one of the vault servers
+* 3 - To login in consul look for master token under aws secrets manager 
+
+# Vault auto unseal example
```

Comment:
You already have a `#` (an `h1`) up above, so this should be `##`... And perhaps it needs a more specific title, as you already mentioned vault auto unseal example at the top.